lib/dal/src/func/binding/management.rs

```diff
@@ -70,93 +69,134 @@ impl ManagementBinding {
             component_create_type,
             component_input_type,
         ) = match ManagementPrototype::prototype_id_for_func_id(ctx, func_id).await? {
-            Some(prototype_id) => match ManagementPrototype::get_by_id(ctx, prototype_id).await? {
```

Comment:
I suggest using the "hide whitespace" option to review this; removing managed_schemas caused us not to need the ManagementPrototype anymore, which caused a reindent.